rclpy/rclpy/executors.py

```diff
@@ -208,6 +208,8 @@ def create_task(self, callback: Union[Callable, Coroutine], *args, **kwargs) ->
         """
         Add a callback or coroutine to be executed during :meth:`spin` and return a Future.
 
+        Created task is queued in the executor in FIFO order.
+
```

Comment:
probably we should add more documentation that says execution order is not guaranteed? so user application should not rely on the execution order?